CSharpLinq/Program.cs

```diff
@@ -0,0 +1,180 @@
+
+            Console.WriteLine("Task 4: Cities that start with 'M' and end with 'd':");
+            foreach (var city in filteredCities)
+            {
+                Console.WriteLine(city);
+            }
+            Console.WriteLine();
+            #endregion
+
+            #region Task5
+            Console.WriteLine("Task 5: Enter a string with words in uppercase and lowercase:");
+            string inputString = Console.ReadLine();
+
+            var words = inputString.Split(' ');
+            var uppercaseWords = words.Where(word => word == word.ToUpper());
+
+            Console.WriteLine("Words in uppercase:");
+            foreach (var word in uppercaseWords)
+            {
+                Console.WriteLine(word);
+            }
+            Console.WriteLine();
+            #endregion
+
+            #region Task6
+            List<int> numbers3 = new List<int> { 1, 2, 3, 4, 2, 5, 6, 3, 4, 7 };
+
+            var distinctNumbers = numbers3.Distinct().Reverse();
+
+            Console.WriteLine("Task 6: Distinct numbers in reverse order:");
+            foreach (var num in distinctNumbers)
+            {
+                Console.WriteLine(num);
+            }
+            Console.WriteLine();
+            #endregion
+
+            #region Task7
+            List<int> numbers4 = new List<int> { 1, 2, 3, 4, 5, 6, 7, 8, 9, 10 };
+
+            Console.WriteLine("Task 7: Enter a number:");
+            int userInput = Convert.ToInt32(Console.ReadLine());
+
+            bool numberExists = numbers4.Contains(userInput);
+
+            if (numberExists)
+            {
+                Console.WriteLine("The number exists in the collection.");
+            }
+            else
+            {
+                Console.WriteLine("The number does not exist in the collection.");
+            }
+            Console.WriteLine();
+            #endregion
+
+            #region Task8
+            List<string> strings = new List<string> { "abc", "defg", "hijklm" };
+
+            int totalLength = strings.Sum(str => str.Length);
+
+            Console.WriteLine("Task 8: Total length of all strings: " + totalLength);
+
+            if (totalLength > 5)
+            {
+                Console.WriteLine("First element: " + strings.First());
+                Console.WriteLine("Last element: " + strings.Last());
+            }
+            Console.WriteLine();
+            #endregion
+
+            #region Task9
+            List<string> colors = new List<string> { "bald", "brown", "yellow" };
+
+            var otterCollection = colors.Select(color => new Otter { Color = color }).ToList();
+
+            Console.WriteLine("Task 9: Otter collection (color property):");
+            foreach (var otter in otterCollection)
+            {
+                Console.WriteLine("Color: " + otter.Color);
+            }
+            Console.WriteLine();
+            #endregion
+
+            #region Task10
+            List<Otter> otters = new List<Otter>
```

Comment:
you can write var otters instead of List<Otter> otters to increase readability :)